styles/app.scss

```diff
@@ -1,5 +1,226 @@
 $fa-font-path: '//netdna.bootstrapcdn.com/font-awesome/4.3.0/fonts';
-
+$BrandPrimary: #3473b5;
```

Comment:
When creating sass variables, stick to dashes between words and all lowercase to stay inline with the linting errors